docs/content/en/latest/references/embed_semantic_search/_index.md

```diff
@@ -0,0 +1,179 @@
+                            {item.title}
+                        </li>
+                    ))}
+                </ul>
+            )}
+        </div>
+    );
+};
+```
+
+### Hook Parameters
+
+| Name        | Type               | Default | Description                                                              |
+| ----------- | ------------------ | ------- | ------------------------------------------------------------------------ |
+| searchTerm  | string             | -       | The search term to use for semantic search                               |
+| objectTypes | GenAIObjectType[]  | []      | A list of object types to search for                                     |
+| deepSearch  | boolean            | false   | Enable deep search to find dashboards by their contents                  |
+| limit       | number             | -       | Maximum number of search results to return                               |
+| backend     | IAnalyticalBackend | -       | Backend instance. Falls back to BackendProvider context if not specified |
+| workspace   | string             | -       | Workspace ID. Falls back to WorkspaceProvider context if not specified   |
+
+### Hook Return Value
+
+The hook returns an object with the following properties:
+
+| Name          | Type                                        | Description                                                                                   |
+| ------------- | ------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| searchStatus  | "idle" \| "loading" \| "error" \| "success" | Current status of the search operation                                                        |
+| searchError   | string                                      | Error message if the search failed                                                            |
+| searchResults | ISemanticSearchResultItem[]                 | Array of search results                                                                       |
+| relationships | ISemanticSearchRelationship[]               | Array of relationships between search results (e.g., dashboard containing a matching insight) |
+
+## Search Result Item Interface
+
+The `ISemanticSearchResultItem` interface represents an item in the search results:
+
+```typescript
+interface ISemanticSearchResultItem {
+    /**
+     * The ID of the item.
+     */
+    id: string;
+
+    /**
+     * The title of the item.
+     */
+    title: string;
+
+    /**
+     * The description of the item.
+     */
+    description?: string;
+
+    /**
+     * The type of the item (e.g., "dashboard", "metric", "insight").
+     */
+    type: string;
+
+    /**
+     * The URL to the item in the GoodData platform.
+     */
+    link?: string;
+
+    /**
+     * Additional metadata about the item.
+     */
+    meta?: Record<string, any>;
+}
+```
+
+[theme provider]: ../../learn/apply_theming/
```

Comment:
If I remember it right, the reference link should be of exact same case, otherwise it will not work. Could you please verify this?